### DISCUSSION TITLE
Raven BD error

### DISCUSSION THREAD

**Discussion by abcdata2010** [Finbuckle/Finbuckle.MultiTenant#556] [General]
Hi,
Am I the only one who has the Raven Db error: Raven limits the number of remote calls that a session is allowed to make as an early warning system? There is a 30-call limit in the Raven Db default setting. I could change this number but I would like to close the Raven Db session after each call and open it again when needed.
I am using the Finbuckle.MultiTenant NPM in the Blazor Server app. I am using the "Transient" in all places of the Finbuckle.MultiTenant.RavenDb.Store but the problem is still here.
Thanks.

**Comment by AndrewTriesToCode** [comment]
Hi, it looks like you might be the first to hit this error. I don't use Raven myself so I can't easily test this.That particular NuGet package is not part of the official Finbuckle packages so I can't vouch for it. If you do find a workaround please let me know.

Thanks,
Andrew
